src/sass/formapp.scss

```diff
@@ -1,4 +1,4 @@
-html, body, #root { height: 100% }
+@import './app-root.scss';
```

Comment:
I think this isn't needed?  since formapp is loaded from index.  These styles would already be in the DOM